frontend/src/components/activity/content/LAThematicArea.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <v-list three-line>
+    <v-list-item v-for="option in activityContent.options().items" :key="option.id" class="px-0">
+      <v-list-item-action>
+        <api-checkbox :id="option.id" fieldname="checked" :uri="option._meta.self" />
```

Comment:
I found a strange bug in these checkboxes. It is not happening on the /controls page. I cannot click the center part of the checkbox, the pointer is wrong and clicking does nothing. I can still click the "glowing" ring around the checkbox to activate and deactivate it.
This happens on both Firefox 84 and Chromium 87 on Ubuntu.

https://user-images.githubusercontent.com/7566995/104821052-70931680-5839-11eb-850e-ec9e3e9b4572.mp4

Reply:
Das Problem tritt bei mir weiterhin auf, sowohl in Firefox als auch in Chromium. Kannst du es reproduzieren?

Reply:
Sorry ich hatte es angeschaut, Änderungen gemacht und plötzlich hat es geklappt und da habe ich es commited. Bin da aber noch nicht fertig.

Reply:
Ich hatte den Bug falsch verstanden. Das Problem ist dass das Diskettensymbol die Checkbox verdeckt. Am besten wird das gerade in #894 gelöst.

Reply:
Weshalb tritt das dann nicht bei anderen Checkboxes auf?

Reply:
Weil sie mehr Platz haben. Sie füllen die ganze Breite aus die ihnen zur Verfügung steht. In der `v-list-item-action` ist die Breite von Vuetify her limitiert.

Dieses Speicher-Status-Icon möchte ich aber sowieso in #894 lösen/ändern.